operators/multiclusterobservability/manifests/base/alertmanager/alertmanager-statefulset.yaml

```diff
@@ -112,7 +112,7 @@ spec:
         - --skip-provider-button=true
         - --openshift-ca=/etc/pki/tls/cert.pem
         - --openshift-ca=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt
-        image: quay.io/open-cluster-management/origin-oauth-proxy:2.0.11-SNAPSHOT-2021-04-29-18-29-17
+        image: quay.io/stolostron/origin-oauth-proxy:2.0.11-SNAPSHOT-2021-04-29-18-29-17
```

Comment:
cannot find this image. I am not sure how does it pass our test. @marcolan018 Do you know why?

Reply:
oh. I got it. Are you disable ci/prow/e2e-kind for release-2.4?

Reply:
We do not enable test in KinD for release branch. Do you think we need to enable it in release branch?

Reply:
oh， for this one. I think it's required to be reverted, to use the image in open-cluster-management.